pkg/storage/chunk/cache/groupcache.go

```diff
@@ -215,6 +224,11 @@ func (c *GroupCache) NewGroup(name string, ct stats.CacheType) Cache {
 	c.wg.Add(1)
 	c.startWaitingForClose()
 
+	cap := c.capacity
+	if cfg.CapacityMB != 0 {
+		cap = cfg.CapacityMB * 1e6 // MB into bytes
+	}
```

Comment:
Hhmm, do we need to keep the `capacity` field here?

Reply:
I would think so. `c.capacity` comes from global groupcache config (when we create single groupcache instance from `common.Groupcache` config).

`Groupcache.capacity` -> global capacity
`group.cacheBytes` -> local to particular group.

We need both.

does that answers your question? or is it something else?. 

nit: I can make both consistent name as `cacheBytes` (in sync with how it's used in original google/mailgun Groupcache package).

Reply:
Ah gotcha :+1: 
Thanks